Controllers/AddWeatherToIntervention.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  AddWeatherToIntervention.swift
+//  Clic&Farm-iOS
+//
+//  Created by Jonathan DE HAAY on 07/09/2018.
+//  Copyright © 2018 Ekylibre. All rights reserved.
+//
+
+import UIKit
+import CoreData
+
+extension AddInterventionViewController {
+
+  // MARK: - Initialization
+
+  func initializeWeatherView() {
```

Comment:
Because you have an array of buttons, I think it's way better to use that array since you do the same for all of those buttons